lib/router.js

```diff
@@ -88,15 +88,22 @@ router.close = function() {
   server = null;
 };
 
+router._routes = { };
```

Comment:
I like the theory behind what they're saying, but I disagree with us changing things:

1. The edge cases they're trying to avoid aren't applicable to us, the only place we touch prototypes in this project is the in-memory handler. 
2. I like to keep things simple.
3. I like to keep things consistent. We don't do this anywhere else in this codebase. Nor do we need to.